### ISSUE TITLE
Feature Request: Cache Hoards while away from server.

### ISSUE THREAD

**Issue by hockeygoalie35** [hoarder-app/hoarder#274]
Hi! Absolutely love this app! I run my hoarder instance locally and don’t expose it outside my network. Is it possible on the iOS app to cache hoards locally on the phone until the user gets home, and sync it? I know that it would be a substantial undertaking to implement.

**Comment by MohamedBassem** [comment]
Yeah, this makes sense as a feature request, but as you mentioned, it's probably going to be a bit tricky to implement. So can't promise it'll happen soon :)

**Comment by hockeygoalie35** [comment]
No promise necessary! Love the app, here to watch it grow!

**Comment by anselor** [comment]
I, too, would like this feature. Thank you.
